mailpoet/tests/integration/Automation/Integrations/MailPoet/Actions/SendEmailActionTest.php

```
@@ -246,10 +246,12 @@ function() use ($args, $controller) {
     $actionScheduler = $this->diContainer->get(ActionScheduler::class);
     $this->assertCount(0, $actionScheduler->getScheduledActions());
     $this->action->run($args, $controller);
-    $actions = array_values($actionScheduler->getScheduledActions());
+    $actions = $actionScheduler->getScheduledActions();
     $this->assertCount(1, $actions);
-    $this->assertSame('mailpoet/automation/step', $actions[0]->get_hook());
-    $this->assertSame([['automation_run_id' => $run->getId(), 'step_id' => 'step-id', 'run_number' => 3]], $actions[0]->get_args());
+    $action = reset($actions);
+    $this->assertNotFalse($action);
+    $this->assertSame('mailpoet/automation/step', $action->get_hook());
```

PHPStan (2.1.51)

[error] Cannot call method get_hook() on ActionScheduler_Action|false.


(method.nonObject)